web3js-ext/src/web3/web3.ts

```diff
@@ -1,13 +1,13 @@
 import Web3, {Bytes, Transaction, Web3Context} from "web3";
-import { signTransaction, SignTransactionResult, privateKeyToAddress } from "web3-eth-accounts";
+import { signTransaction, SignTransactionResult, privateKeyToAddress, Web3Account, Wallet, decrypt, } from "web3-eth-accounts";
 import { bytesToHex } from "web3-utils";
-import { DataFormat, DEFAULT_RETURN_FORMAT } from "web3-types";
+import { Address, HexString, DataFormat, DEFAULT_RETURN_FORMAT, KeyStore } from "web3-types";
 import { SendTransactionOptions } from "web3-eth";
 import _ from "lodash";
 
 import { prepareTransaction } from "./klaytn_tx";
 import { klay_sendSignedTransaction } from "./send_transaction";
-import { signTransactionAsFeePayer } from "./account";
+import { privateKeyToAccountWithContext as privateKeyToAccountWithContext, signTransactionAsFeePayer, createWithContext, recoverTransactionWithKlaytnTx } from "./account";
```

Comment:
```suggestion
import { privateKeyToAccountWithContext, signTransactionAsFeePayer, createWithContext, recoverTransactionWithKlaytnTx } from "./account";
```

Reply:
Updated in https://github.com/klaytn/web3klaytn/pull/522/commits/5d13af491f3abb182a3c7bbf23833af9ac746ba7. 
Thanks.